modern/src/reports/RouteReportPage.js

```diff
@@ -26,35 +27,62 @@ const Filter = ({ setItems }) => {
 };
 
 const RouteReportPage = () => {
+  const distanceUnit = useAttributePreference('distanceUnit');
+
+  const columns = [{
+    headerName: t('positionFixTime'),
+    field: 'fixTime',
+    type: 'dateTime',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'fixTime'),
+  }, {
+    headerName: t('positionLatitude'),
+    field: 'latitude',
+    type: 'number',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'latitude'),
+  }, {
+    headerName: t('positionLongitude'),
+    field: 'longitude',
+    type: 'number',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'longitude'),
+  }, {
+    headerName: t('positionSpeed'),
+    field: 'speed',
+    type: 'number',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'speed'),
+  }, {
+    headerName: t('positionAddress'),
+    field: 'address',
+    type: 'string',
+    flex: 1,
+    valueFormatter: ({ value }) => formatPosition(value, 'address'),
+  }, {
+    headerName: t('positionIgnition'),
+    field: 'ignition',
+    type: 'boolean',
+    flex: 1,
+    valueFormatter: ({ getValue }) => getValue('attributes').ignition ? 'Yes' : 'No',
+  }, {
+    headerName: t('deviceTotalDistance'),
+    field: 'totalDistance',
+    type: 'number',
+    hide: true,
+    flex: 1,
+    valueFormatter: ({ getValue }) => formatDistance(getValue('attributes').totalDistance, distanceUnit),
```

Comment:
What is `getValue` and how does grid get value for this column?

Reply:
In DataGrid we pass rows as group of objects in an array.  getValue is a function which is used to get the value from current object by passing a key. In this case we pass the key 'attributes' which return attribute object.

Reply:
I feel like the right solution is to use [valueGetter](https://material-ui.com/components/data-grid/rendering/#value-getter). Otherwise wouldn't ordering/filtering not work if you don't get proper value?

Reply:
Ok. I'll do changes accordingly.